olp-cpp-sdk-core/tests/cache/DefaultCacheTest.cpp

```diff
@@ -219,3 +220,57 @@ TEST(DefaultCacheTest, AlreadyInUsePath) {
   olp::cache::DefaultCache cache2(settings);
   ASSERT_EQ(olp::cache::DefaultCache::OpenDiskPathFailure, cache2.Open());
 }
+
+class ProtectedCacheTest : public ::testing::Test {
+ protected:
+  void SetUp() override {
+    cache_path = olp::utils::Dir::TempDirectory() + "/unittest";
+
+    // Create cache
+    olp::cache::CacheSettings settings;
+    settings.disk_path_mutable = cache_path;
+    olp::cache::DefaultCache cache(settings);
+    ASSERT_EQ(olp::cache::DefaultCache::Success, cache.Open());
+    ASSERT_TRUE(cache.Clear());
+    std::string key1DataString{"this is key1's data"};
```

Comment:
```suggestion
    std::string key1_data_string{"this is key1's data"};
```
Refer t o google C++ code style